src/Maestro/maestro-angular/maestro-angular.proj

```diff
@@ -67,4 +67,8 @@
 
   <Target Name="GetTargetPath">
   </Target>
+  
+  <Target Name="Clean">
+    <Delete Files="@(BuildOutput)" />
```

Comment:
I think technically this doesn't "clean" everything. It deletes the marker file that prevents build from happening, but it doesn't clean the actual output files. Its probably fine though.

Reply:
I'm not sure what else to clean, because this thing doesn't understand what it did, it's just delegating to npm, right?  Is there a clean command I should run?  I guess I could also override the "Rebuild" directory directly, but that'll be trickier.

Reply:
Yea, this is probably fine. We can't really tell npm to clean, we would just have to delete all the stuff.

Reply:
And this will make rebuild do the right thing, because if this file is gone we tell npm to build again.